torch/lib/c10d/FileStore.cpp

```diff
@@ -21,6 +28,41 @@
     throw std::system_error(errno, std::system_category(), ##__VA_ARGS__); \
   }
 
+#ifdef _WIN32
+#define LOCK_EX 0x00000001
+#define LOCK_SH 0x00000010
+#define LOCK_UN 0x00000100
+
+int flock_(int fd, int op)
+{
```

Comment:
to stay consistent with the rest of the code. This `{` should belong to the line above.